### ISSUE TITLE
Accessibility: My Account orders table should have row-scoped headers in addition to column headers

### ISSUE THREAD

**Issue by amberhinds** [woothemes/woocommerce#43629]
### Prerequisites

- [X] I have carried out troubleshooting steps and I believe I have found a bug.
- [X] I have searched for similar bugs in both open and closed issues and cannot find a duplicate.

### Describe the bug

The orders table on the My Account page currently only has headers for the columns. The Order number (first column) also functions as a header and should be in a TH tag with a scope of row rather than a TD tag.

Relevant code:
https://github.com/woocommerce/woocommerce/blob/758df4854d7b627fc26eb5c0860dc492dea6a06e/plugins/woocommerce/templates/myaccount/orders.php#L26

### Expected behavior

1. All of the column headers need scope="col" added to them.
2. Change the order number cell to a `<th> `with scope="row".

This will allow screen readers to more clearly connect the order number with the other cells in the table.

### Actual behavior

Order numbers are not clearly connected to the date, status, total, and actions.

### Steps to reproduce

Inspect the orders table or see code linked above.

### WordPress Environment

N/A

### Isolating the problem

- [X] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [X] This bug happens with a default WordPress theme active, or [Storefront](https://woo.com/products/storefront/).
- [X] I can reproduce this bug consistently using the steps above.

**Comment by jeffpaul** [comment]
Note that @Manussakis has a PR up for this issue: https://github.com/woocommerce/woocommerce/pull/49201
